tests/integration-tests/tests/storage/test_fsx_lustre.py

```diff
@@ -98,3 +108,50 @@ def _test_export_path(remote_command_executor, mount_dir, bucket_name):
     )
     result = remote_command_executor.run_remote_command("cat ./file_to_export")
     assert_that(result.stdout).is_equal_to("Exported by FSx Lustre")
+
+
+def _test_data_repository_task(remote_command_executor, mount_dir, bucket_name, fsx_fs_id, region):
+    logging.info("Testing fsx lustre data repository task")
+    remote_command_executor.run_remote_command(
+        "echo 'Exported by FSx Lustre' > {mount_dir}/file_to_export".format(mount_dir=mount_dir)
+    )
+
+    # set file permissions
+    remote_command_executor.run_remote_command(
+        "sudo chmod 777 {mount_dir}/file_to_export && sudo chown 6666:6666 {mount_dir}/file_to_export".format(
+            mount_dir=mount_dir
+        )
+    )
+
+    fsx = boto3.client("fsx", region_name=region)
+    task = fsx.create_data_repository_task(
+        FileSystemId=fsx_fs_id, Type="EXPORT_TO_REPOSITORY", Paths=["file_to_export"], Report={"Enabled": False}
+    ).get("DataRepositoryTask")
+
+    status = task.get("Lifecycle")
+    task_id = task.get("TaskId")
+
+    while status in ["PENDING", "EXECUTING"]:
+        logging.info("Data Export Task {task_id}: {status}".format(task_id=task_id, status=status))
+        status = fsx.describe_data_repository_tasks(TaskIds=[task_id]).get("DataRepositoryTasks")[0].get("Lifecycle")
+        time.sleep(5)
+
+    assert_that(status).is_equal_to("SUCCEEDED")
+
+    remote_command_executor.run_remote_command(
+        "aws s3 cp s3://{bucket_name}/export_dir/file_to_export ./file_to_export".format(bucket_name=bucket_name)
+    )
+    result = remote_command_executor.run_remote_command("cat ./file_to_export")
+    assert_that(result.stdout).is_equal_to("Exported by FSx Lustre")
```

Comment:
minor: it looks like you use this string a few places in your test. it might be nice to just define a constant for it.

Reply:
done